canonical-web-and-design/ubuntu.com
# Get attempts for each course from EDX

## Done
Get attempts for each course from EDX to work out the course status.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cube/microcerts
- Make sure the page still renders and you courses status are correct
